programs/survival/renewable_elytra_trade.sc

```diff
@@ -0,0 +1,45 @@
+// Customize the trade
+
+global_trade = encode_nbt(
+	{
+		'sell' -> {'id' -> 'minecraft:elytra', 'Count'->1},
+		'buy' -> {'id' -> 'minecraft:emerald', 'Count' -> 40},
+		'buyB' -> {'id' -> 'minecraft:phantom_membrane', 'Count' -> 5},
+		'uses' -> 0,
+		'demand' -> -20,
+		'priceMutiplier' -> 0.0,
+		'specialPrice' -> 0,
+		'rewardExp' -> 1, // boolean (gives/doesn't give xp)
+		'xp' -> 1, // count
+	}
+);
+
+//set the changes of the trade appearing in %
+global_chances = 100;
+
+// Customize the conditions
+_add_trade(e, p) -> (
+	nbt = e~'nbt';
+	if(
+		//e~'dimension'=='the_end' && //villager needs to be in the end
```

Comment:
Adding a config to enable this requirement(?)

Reply:
All conditions should be configurable outside the main code, you mean? I just left "in the end required" off by default because I thought it'd be a good example of how to turn off conditions. I can move them all to a config above.

Reply:
I allways prefer to have a way to change config from the chat, wirhout editing the code.

Reply:
Will add this a, long with better documentation on how to customize the app.